exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetReaderConfig.java

```diff
@@ -50,6 +50,7 @@
   private boolean enableTimeReadCounter = false;
   private boolean autoCorrectCorruptedDates = true;
   private boolean enableStringsSignedMinMax = false;
+  private long timeoutPerRunnableInMsec = 15_000;
```

Comment:
Since the 'owner' of the default should be ExecConstants, we should avoid this default value.